nvme.c

```diff
@@ -5888,9 +5888,10 @@ static int set_feature(int argc, char **argv, struct command *cmd, struct plugin
 							cfg.file, strerror(errno));
 					return -EINVAL;
 				}
+				err = read(ffd, (void *)buf, cfg.data_len);
+			} else {
+				err = read(STDIN_FILENO, (void *)buf, cfg.data_len);
```

Comment:
Pre-existing, but doesn't seem like these `void *` casts should be necessary, as that's already the type of `buf`